src/graphql/main/schema.graphql

```diff
@@ -309,6 +310,22 @@ type Coordinates {
   latitude: Float!
 }
 
+type BtcUsdPrice {
+  timestamp: Timestamp!
+  price: Float!
```

Comment:
Don't we have a more specific type? Like USDPerSat or something like that? `Float` is not a good type to use.

Reply:
Changed to use a price object with base/offset